internal/engine/session_integration_test.go

```diff
@@ -584,50 +617,31 @@ func TestNewOrchestraClientMaybeLookupBackendsFailure(t *testing.T) {
 	if testing.Short() {
 		t.Skip("skip test in short mode")
 	}
+	errMocked := errors.New("mocked error")
 	sess := newSessionForTestingNoLookups(t)
-	ctx, cancel := context.WithCancel(context.Background())
-	cancel() // fail immediately
-	client, err := sess.NewOrchestraClient(ctx)
-	if !errors.Is(err, ErrAllProbeServicesFailed) {
-		t.Fatal("not the error we expected")
+	sess.testMaybeLookupBackendsContext = func(ctx context.Context) error {
+		return errMocked
+	}
+	client, err := sess.NewOrchestraClient(context.Background())
+	if !errors.Is(err, errMocked) {
+		t.Fatal("not the error we expected", err)
```

Comment:
This test needed adjusting because now MaybeLookupBackends immediately fails w/ a cancelled context